def main(params):
  raise Exception("error")
